lib/simplifyExpression/distribute/index.js

```diff
@@ -1,49 +1,48 @@
 'use strict';
 
-const clone = require('./util/clone');
-const collectAndCombineSearch = require('./collectAndCombine/collectAndCombineSearch');
-const evaluateArithmeticSearch = require('./evaluateArithmeticSearch');
-const MathChangeTypes = require('./MathChangeTypes');
-const Negative = require('./util/Negative');
-const NodeCreator = require('./util/NodeCreator');
-const NodeStatus = require('./util/NodeStatus');
-const NodeType = require('./util/NodeType');
-const rearrangeCoefficient = require('./simplifyBasics/rearrangeCoefficient');
-const TreeSearch = require('./util/TreeSearch');
+const clone = require('../../util/clone');
+const collectAndCombineSearch = require('../collectAndCombine');
```

Comment:
I'm wondering if we should name the folders with Search so it's clear that the index.js is a search?

is it weird for the thing we require to be different from the name
like `const A = require('B')` instead of `const A = require('A')`